hat/app/org/hatdex/hat/api/controllers/Authentication.scala

```diff
@@ -494,7 +494,7 @@ class Authentication @Inject() (
                         )
                         Done
                     }
-                  mailer.passwordChanged(token.email)
+                  mailer.hatClaimed(token.email, user.name)
```

Comment:
@augustinas a bit of help here, `user.name` is it going to be `something.dataswift.dev`? It's a `HatUser` type

Reply:
It's actually supposed to be `https://hat.dataswift.dev` to lead them to log in page. Or the sign in page in general. So this has to change definitely I am not sure how to get the hat address. Or actually redirect them to PDA Auth to log in? 🤷‍♂️

Reply:
Link can have redirect to the authentication service or direct to PDA's password screen. The question I would ask is what application ID it should carry in the redirect? What is the intended action after the button click?